pallets/identity/src/lib.rs

```diff
@@ -458,10 +458,22 @@ decl_module! {
                 |batch_claim_item| <DidRecords>::contains_key(batch_claim_item.target)),
                 Error::<T>::DidMustAlreadyExist);
 
+            let cdd_providers = T::CddServiceProviders::get_members();
```

Comment:
as a minor optimization, this line can be moved inside `if cdd_count > 0`.

Reply:
alright will make the change.